src/_includes/partials/anonymous-contact-form.njk

```diff
@@ -0,0 +1,22 @@
+<header class="content__header">
+  <h1 class="title page__title">
+    {{ title }}
+  </h1>
+</header>
+
+<article class="bg copy page__content">
+  {{ content | markdown | safe }}
+
+  <br>
+
+  <form name="coc-report" method="POST" netlify-honeypot="bot-field" data-netlify="true" action="/success-message" >
+    <p>
+      <label>Message:</label>
+      <br> 
+      <textarea name="report" cols="40" rows="10" id="message" required="true"></textarea>
+    </p>
```

Comment:
So we just need to add an input whose name matches the value of the `data-honeypot` attribute, a `.hidden` class to the CSS and we're good 👍

```html
<p class="hidden">
  <label>Don’t fill this out if you're human: 
    <input name="bot-field" />
  </label>
</p>
```